packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -224,6 +234,22 @@ class MessageList extends Widget<Properties> {
     }
   }
 
+  _isTimeoutExceeded(lastMessage: Message, newMessage: Message): boolean {
+    const lastMessageTimestamp = lastMessage?.timestamp;
+    const newMessageTimestamp = newMessage?.timestamp;
+
+    if (!lastMessageTimestamp || !newMessageTimestamp) {
+      return false;
+    }
+
+    const lastMessageTimestampInMs = new Date(lastMessageTimestamp).getTime();
+    const newMessageTimestampInMs = new Date(newMessageTimestamp).getTime();
```

Comment:
Why is it required to call getTime()? There are no difficulties using the '-' and '+' operations with the date format. For example:
`new Date(2024, 10, 9) - new Date(2024, 10, 10) = -86400000`

Reply:
Okay, I get it. [9901c1004acbc319ac5547e3c7df9fe5a0427569]